### PR TITLE
Add kabyle translations

### PR DESCRIPTION
**Pull request by Azwaw1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <p align="center"><img src="https://user-images.githubusercontent.com/56656996/99621835-b2d8dc80-29dd-11eb-8183-987e80f8b3a7.png"></p>
 <h1 align="center">DDE Store</h1>
 <h3 align="center">An app store for DDE built with DTK</h3>
-<p align="center"><a href="/translations/README/README.es.md">🇪🇸 Español</a> | <a href="/translations/README/README.de.md">🇩🇪 Deutsch</a> | <a href="/translations/README/README.tr.md">🇹🇷 Türkçe</a> | <a href="/translations/README/README.ar.md">🇸🇦 العربية</a></p>
+<p align="center"><a href="/translations/README/README.es.md">🇪🇸 Español</a> | <a href="/translations/README/README.de.md">🇩🇪 Deutsch</a> | <a href="/translations/README/README.kab.md">Taqbaylit</a> | | <a href="/translations/README/README.tr.md">🇹🇷 Türkçe</a> |<a href="/translations/README/README.ar.md">🇸🇦 العربية</a></p>
```
Comment (ghost) [inline README.md]: Is there any particular flag emoji that could be used for the link?

**Comment by ghost** [comment]
Thank you, this looks great! Would you mind adding a name and description translation to the .desktop file?

File: README.md (hunk 1 above)
Comment (Aksel103) [reply README.md]: @dekzi 
here is the kabyle flag:https://fr.wikipedia.org/wiki/Fichier:Flag-kabyle.svg , but unfortunately there is no emoji with this flag, not yet and I could not create one from the flag.

Comment (ghost) [reply README.md]: > @dekzi
> here is the kabyle flag:https://fr.wikipedia.org/wiki/Fichier:Flag-kabyle.svg , but unfortunately there is no emoji with this flag, not yet and I could not create one from the flag.

Would the Algerian flag be okay for recognizably? 🇩🇿

Comment (Aksel103) [reply README.md]: yes, it will be okay

**Comment by Azwaw1** [comment]
@dekzi I don't find the .desktop file, would you give me the link to this file?

**Comment by SelyanKab** [comment]
@dekzi I don't find the .desktop file, would you give me the link to this file?

**Comment by ghost** [comment]
> @dekzi I don't find the .desktop file, would you give me the link to this file?

`resources/dde-store.desktop`

**Comment by Azwaw1** [comment]
@dekzi  Done.

**Comment by ghost** [comment]
Thanks!

**Comment by Azwaw1** [comment]
@dekzi thanks to you.

**Pull request merged**